artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/addressSendMessage.js

```diff
@@ -87,15 +91,16 @@ var Artemis;
             </form>
 
             <p>
-                <button type="button" class="btn btn-primary artemis-send-message-button" ng-click="$ctrl.message.sendMessage($ctrl.message.durable)">Send message</button>
+                <button type="button" class="btn btn-primary artemis-send-message-button" ng-click="$ctrl.message.sendMessage($ctrl.message.durable, $ctrl.message.createUserId)">Send message</button>
             </p>
             <script type="text/ng-template" id="send-message-instructions.html">
             <div>
                 <p>
                     This page allows you to send a message to the chosen address. The message will be of type <code>text</code>
                     message and it will be possible to add headers to the message. The sending of the message will be authenticated
                     using the username and password set ion <code>preferences</code>, if this is not set then these will
-                    be null.
+                    be null. The <code>durable</code> checkbox makes the message persistent and the <code>create user Id</code> will auto generate a
+                    UUID user for the message which for a JMS Message is the JMS Message ID.
```

Comment:
_"the \<code\>create user Id\</code\> will"_ seems like it should have _checkbox_ or something else added to complete.

_"a UUID user for"_ seems like it is missing _id_.